src/Apprenticeships/SFA.DAS.Apprenticeships.Api/Extensions/MapperExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿using SFA.DAS.Apprenticeships.Api.Models;
+using SFA.DAS.Apprenticeships.Application.Notifications.Handlers;
+using PostCreateApprenticeshipPriceChangeRequest = SFA.DAS.SharedOuterApi.InnerApi.Requests.Apprenticeships.PostCreateApprenticeshipPriceChangeRequest;
+
+namespace SFA.DAS.Apprenticeships.Api.Extensions
+{
+    // Code review topic for discussion, can we 
```

Comment:
Not sure what you wanted to discuss here but this class seems good to me. Were you going to suggest AutoMapper?